File: Gettingstarted-sample-c#/Fontcombobox_sample/Form1.cs
using Syncfusion.Windows.Forms.Tools;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Fontcombobox_sample
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            //Initialize the FontComboBox control.
            FontComboBox fontComboBox = new FontComboBox();
            fontComboBox.Size = new Size(150, 40);
            fontComboBox.Location = new Point(100, 100);
            fontComboBox.VisualStyle = ThemedComboBoxStyles.Office2016Colorful;
            fontComboBox.UseAutoComplete = true;
            fontComboBox.SelectedItem = "Arial";
            this.Controls.Add(fontComboBox);
        }
    }
}

File: Gettingstarted-sample-vb/Fontcombobox_sample/Form1.vb
Imports Microsoft.VisualBasic
Imports Syncfusion.Windows.Forms.Tools
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace Fontcombobox_sample
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			'Initialize the FontComboBox control.
			Dim fontComboBox As New FontComboBox()
			fontComboBox.Size = New Size(150, 40)
			fontComboBox.Location = New Point(100, 100)
			fontComboBox.VisualStyle = ThemedComboBoxStyles.Office2016Colorful
			fontComboBox.UseAutoComplete = True
			fontComboBox.SelectedItem = "Arial"
			Me.Controls.Add(fontComboBox)
		End Sub
	End Class
End Namespace
